Resources/doc/index.rst

```diff
@@ -184,4 +188,45 @@ example:
 
 see :ref:`Annotated Routes Activation<frameworkextra-annotations-routing-activation>` for more details.
 
+PSR-7 support
+-------------
+
+SensioFrameworkExtraBundle provides support for HTTP messages interfaces defined
+in `PSR-7`_. It allows to inject instances of :class:`Psr\\Http\\Message\\ServerRequestInterface`
+and to return instances of :class:`Psr\\Http\\Message\\ResponseInterface` in controllers.
```

Comment:
Using the `class` for external class will not work as their API docs are not rendered on symfony.com.